src/views/layouts-wallet/TheSwapLayout.vue

```diff
@@ -35,13 +35,7 @@ export default {
   },
   mixins: [handlerAnalytics],
   beforeRouteLeave(to, from, next) {
-    if (to.name === ROUTES_WALLET.NETWORK.NAME) {
-      this.trackSwap('switchingNetworkOnSwap');
-    }
-    if (
-      to.name !== ROUTES_WALLET.NETWORK.NAME &&
-      to.name !== ROUTES_WALLET.SWAP.NAME
-    ) {
+    if (to.name !== ROUTES_WALLET.SWAP.NAME) {
```

Comment:
Route doesn't change anymore.  Watch the "isOpenNetworkOverlay" instead, check if current route is swap, then fire event.